concepts/best-practices-concept.md

```diff
@@ -80,6 +80,16 @@ By default, unknown members are not returned by Microsoft Graph. If, however, yo
 
 >**Note:** If your application is prepared to handle unknown enum members, it should opt-in by using an HTTP *prefer* request header: `Prefer: include-unknown-enum-members`.
 
+### Logging for investigation
```

Comment:
There's already a bullet item and a section about supportability at the end of the doc.  Please move it to that section and reorganize this as part of that section.

Reply:
Done.